CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 - Diagnostics related feature requests and improvements [#5951](https://github.com/dotnet/vscode-csharp/issues/5951)
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
+# 2.95.x
```

Comment:
Is this right? It used to be the snap created the blank next-version-number section automatically, but there wasn't one, but I'm pretty sure the 2.94.x ship has sailed. But I was away for a week so who knows. Let me know :)

Reply:
No prerelease last week